sources/packages/backend/apps/api/src/route-controllers/atbc/atbc.students.controller.ts

```diff
@@ -45,18 +47,17 @@ export class ATBCStudentController extends BaseController {
     const student = await this.studentService.getStudentById(
       studentUserToken.studentId,
     );
-    // Check the PD status in DB. Student should only be allowed to check the PD status once.
-    // studentPDSentAt is set when student apply for PD status for the first.
-    // studentPDVerified is null before PD scheduled job update status.
-    // studentPDVerified is true if PD confirmed by ATBC or is true from sfas_individual table.
-    // studentPDVerified is false if PD denied by ATBC.
+    // To apply for a PD, SIN validation must be completed for the student and
+    // not applied for PD already.
```

Comment:
@dheepak-aot , can you confirm this? In our grooming, I remember saying that the student can apply for PD multiple times, especially for the PPD

Reply:
Thanks for bring this up. 
I had a quick chat with @JasonCTang on the same and got to know that, as per the current understanding, student requests disability status only once in our application. In case they want to apply again because of something happened to them later after losing their status, they update ATBC directly and ATBC will be updating the student disability status which we read on daily basis through their response API.
I will update this information to the ticket.

Reply:
Thanks @dheepak-aot for the details. Ya, adding it to the ticket will make it easier for future development